mpas_analysis/ocean/sst_timeseries.py

```diff
@@ -1,23 +1,47 @@
 import xarray as xr
 import pandas as pd
 import datetime
+import os.path
 
 from ..shared.mpas_xarray.mpas_xarray import preprocess_mpas, remove_repeated_time_index
 
 from ..shared.plot.plotting import timeseries_analysis_plot
 
+from ..shared.io import StreamsFile
+
 def sst_timeseries(config):
+    """
+    Performs analysis of the time-series output of sea-surface temperature
+    (SST).
 
+    Author: Xylar Asay-Davis, Milena Veneziani
+    Last Modified: 10/27/2016
+    """
     # Define/read in general variables
     print "  Load SST data..."
-    indir = config.get('paths','archive_dir_ocn')
-    infiles = '%s/am.mpas-o.timeSeriesStats.????-??*nc'%indir
+    # read parameters from config file
+    indir = config.get('paths', 'archive_dir_ocn')
+
+    streams_filename = config.get('input', 'ocean_streams_filename')
+    if not os.path.isabs(streams_filename):
+       # only the file name was given, not the full path, so add indir
+       streams_filename = '{}/{}'.format(indir, streams_filename)
+    streams = StreamsFile(streams_filename, streamsdir=indir)
+
+    # get a list of timeSeriesStats output files from the streams file,
+    # reading only those that are between the start and end dates
+    startDate = config.get('time', 'timeseries_start_date')
+    endDate = config.get('time', 'timeseries_end_date')
+    infiles = streams.readpath('timeSeriesStatsOutput', 'filename_template',
+                               startDate=startDate, endDate=endDate)
+    print 'Reading files {} through {}'.format(infiles[0],infiles[-1])
+
 
     casename = config.get('case','casename')
     ref_casename_v0 = config.get('case','ref_casename_v0')
     indir_v0data = config.get('paths','ref_archive_v0_ocndir')
 
-    compare_with_obs = config.getboolean('sst_timeseries','compare_with_obs')
+    #compare_with_obs = config.getboolean('sst_timeseries','compare_with_obs')
```

Comment:
could we please leave this in for the moment? I am not sure whether we may need it (we have it in the seaice_modelvsobs script for sure).